finance-portal/templates/secrets.yaml

```diff
@@ -21,3 +21,18 @@ data:
   JASPER_USER: {{ .Values.config.jasperUser | b64enc }}
   JASPER_PASSWORD: {{ .Values.config.jasperPassword | b64enc }}
 
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "finance-portal.fullname" . }}-certs
+  labels:
+    app.kubernetes.io/name: {{ include "finance-portal.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+data:
+  first_key: {{ required "The JWS key is required to validate the JWS header" .Values.creds.first_key | b64enc }}
+  second_key: {{ required "The JWS key is required to validate the JWS header" .Values.creds.second_key | b64enc }}
+  third_key: {{ required "The JWS key is required to validate the JWS header" .Values.creds.third_key | b64enc }}
```

Comment:
I think probably all of this should be wrapped in `{{- if .Values.JWSKeys }} .. content .. {{- end }}`